packages/client/src/views/RecordAudit/utils.ts

```diff
@@ -483,9 +479,10 @@ export function getStatusLabel(
   }
   if (
     regStatus === RegStatus.Declared &&
-    performedBy?.id === loggedInUser?.userMgntUserID &&
-    loggedInUser?.systemRole &&
-    FIELD_AGENT_ROLES.includes(loggedInUser.systemRole)
+    performedBy?.id === loggedInUser?.userMgntUserID
+    // @TODO: How do we handle this specific case with scopes?
+    // && loggedInUser?.systemRole &&
+    // FIELD_AGENT_ROLES.includes(loggedInUser.systemRole)
   ) {
     return intl.formatMessage(recordAuditMessages.sentNotification)
   }
```

Comment:
In RecordAudit we `getStatusLabel`. If the role has been field agent and the action is performed by me, it's "sentNotification" -message. How would we handle this with scopes @jpye-finch

Reply:
I think we can remove this requirement. 

In doing so I assume the status would be Incomplete? 

The copy for this status is being reconsidered at the moment